tests/bluetooth/host/buf/cmake.txt

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
we should use this principle.

It has been commented here:
https://github.com/zephyrproject-rtos/zephyr/pull/40073#issuecomment-1041451330

unfortunately I was not invited to that party.

Blocking this PR for time being and will provide a proper prosal after taking closer look at this code.